### PR TITLE
Implement CRUD API

### PR DESCRIPTION
**Pull request by eric1052**
Trello Ticket: https://trello.com/c/6B9OdDYy

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -23,4 +23,4 @@ coverage
 # Compiled Dirs (http://nodejs.org/api/addons.html)
 build/
 dist/
-
+public/bundle.js
```
Comment (Implausibility) [inline .gitignore:26]: Newline.

File: package.json [hunk 2]
```diff
@@ -7,14 +7,14 @@
...
     "test": "jest server/test/ --detectOpenHandles"
   },
   "dependencies": {
     "axios": "^0.19.2",
-    "body-parser": "^1.19.0",
     "css-loader": "^3.5.3",
+    "dotenv": "^8.2.0",
     "express": "^4.17.1",
     "faker": "^4.1.0",
     "jquery": "^3.5.1",
```
Comment (Implausibility) [inline package.json:20]: You changed the use of jQuery to Javascript's built-in `fetch`, so you could uninstall jQuery as a dependency.

File: README.md [hunk 3]
```diff
@@ -1 +1,46 @@
...
+```
+
+## CRUD Operations
+
+| HTTP Verb |           Endpoint          |            Action             |
+|-----------| --------------------------- | ----------------------------- |
+| **POST**  |         /items/             |   CREATE a new item into DB   |
+| **GET**   |       /items/:id            |   READ item based on ID       |
+| **PUT**   |       /items/:id            |   UPDATE one image path       |
+| **DELETE**|       /items/:id            |   DELETE item based on ID     |
```
Comment (Implausibility) [inline README.md:46]: Love this formatting.

File: package.json (hunk 2 above)
Comment (eric1052) [reply package.json:20]: Good point - will fix before merging!

**Pull request merged**